### DISCUSSION TITLE
Carbon::setTestNow() not working with Livewire

### DISCUSSION THREAD

**Discussion by IllesAprod** [livewire/livewire#5628] [1. Help]
Hi! I have a TALL application with a complex UI showing different data based on the day you are watching the UI. 

I would like to manually test the UI (in local dev environment) by setting Carbon::setTestNow() to a past date (in AppServiceProvider), but livewire keeps saying "The page has expired"

Does anyone have a workaround for this?

Thanks!

**Comment by joshhanley** [comment]
@IllesAprod using `setTestNow()` works for me but you should probably be setting it in your test, not in your `AppServiceProvider`.

Can you share your code?
